Automattic/wp-calypso
# Update reader stream styles for main background and card layout.

Related to # pe7F0s-13p-p2 

## Proposed Changes

* Updates the background for reader streams to be as proposed in p2 link above. Also updates the cards as suggested in the p2. Also adds some specific classnames and updates various other styles to accommodate these changes.

<img width="1385" alt="Screenshot 2023-08-01 at 1 36 30 PM" src="https://github.com/Automattic/wp-calypso/assets/28742426/7a34ec9c-a71f-47c8-aacc-085557eb37fd">

## Testing Instructions



* Run this branch of calypso
* visit the various reader stream pages.
* verify the new neutral-0 background is in place
* verify cards have good padding and layout
* search for any visual regressions as a result of these changes, we have handled a few here already. Often they are small areas of white background.
* smoke test non-stream and non-reader pages for style bleed.
* test logged-out reader pages (/read/search, /tags (should be unchanged), /tag/{tagName}, /discover)

## Pre-merge Checklist
